pyquil/api/_qpu.py

```diff
@@ -288,15 +329,20 @@ def _resolve_memory_references(self, expression: Expression) -> Union[float, int
             right = self._resolve_memory_references(expression.op2)
             return expression.fn(left, right)
         elif isinstance(expression, Function):
-            return expression.fn(self._resolve_memory_references(expression.expression))
+            return expression.fn(
+                self._resolve_memory_references(expression.expression))
         elif isinstance(expression, Parameter):
-            raise ValueError(f"Unexpected Parameter in gate expression: {expression}")
+            raise ValueError(
+                f"Unexpected Parameter in gate expression: {expression}")
```

Comment:
unnecessary?

Reply:
This and the other changes in this file are a result of a probably-unintentional run of `yapf` on save